nixnet/_ctypedefs.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import ctypes
+
+
+class char(ctypes.c_char):  # NOQA

```

Comment:
Look to see if we can limit `NOQA` to the part that we are intentionally violating